README.md

```diff
@@ -1,94 +1,89 @@
-to bridge from one transport to another.
+## up-streamer
 
-## Overview
+Generic, pluggable uStreamer that should be usable in most places we need
+to write a uStreamer application to bridge from one transport to another.
 
-Implementation of the uProtocol's uStreamer specification in Rust.
+Reference its README.md for further details.
 
-### Visual Breakdown
+## up-linux-streamer
 
-```mermaid
-sequenceDiagram 
-    participant main thread
-    participant TransportForwarder - Foo
-    participant TransportForwarder - Bar
-    participant UPClientFoo owned thread / task
-    participant UPClientBar owned thread / task
+Concrete implementation of a uStreamer as a binary.
 
-    main thread->>main thread: let utransport_foo: Arc<dyn UTransport> = Arc::new(UPClientFoo::new())
-    main thread->>main thread: let local_authority = ...
-    main thread->>main thread: let local_endpoint = Endpoint::new(local_authority.clone(), utransport_foo.clone())
+Reference its README.md for further details.
 
-    main thread->>main thread: let utransport_bar: Arc<dyn UTransport> = Arc::new(UPClientBar::new())
-    main thread->>main thread: let remote_authority = ...
-    main thread->>main thread: let remote_endpoint = Endpoint::new(remote_authority.clone(), utransport_bar.clone())
-  
-    main thread->>main thread: let ustreamer = UStreamer::new()
+## Building
 
-    main thread->>main thread: ustreamer.add_forwarding_rule(local_endpoint, remote_endpoint)
-    main thread->>TransportForwarder - Foo: launch TransportForwarder - Foo
-    activate TransportForwarder - Foo
-    main thread->>UPClientFoo owned thread / task: (within ustreamer.add_forwarding_rule()) <br> local_endpoint.transport.lock().await.register_listener <br> (uauthority_to_uuri(remote_endpoint.authority), forwarding_listener).await
-    activate UPClientFoo owned thread / task
+Included is a dependency on [up-transport-vsomeip-rust](https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust) in order to communicate via SOME/IP that imposes some additional build considerations.
 
-    main thread->>main thread: ustreamer.add_forwarding_rule(remote_endpoint, local_endpoint)
-    main thread->>TransportForwarder - Bar: launch TransportForwarder - Bar
-    activate TransportForwarder - Bar
-    main thread->>UPClientBar owned thread / task: (within ustreamer.add_forwarding_rule()) <br> remote_endpoint.transport.lock().await.register_listener <br> (uauthority_to_uuri(local_endpoint.authority), forwarding_listener).await
-    activate UPClientBar owned thread / task
+Please reference the documentation for [vsomeip-sys](https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust/tree/main/vsomeip-sys) and note
+* the build requirements for vsomeip in the linked documentation in the COVESA repo
+* the environment variables which must be set
 
-    loop Park the main thread, let background tasks run until closing UStreamer app
+## Running examples
```

Comment:
shouldn't this better be in a README in the examples folder?

Reply:
Okay, I could see that.

Reply:
Done